lblod/db-cleanup-service
# Revamp service

# Description

DL-5601

This PR updates this service to latest dependencies and introduces a mutex lock for scheduling jobs.

For now cronPatterns set in the migrations are purely configured as example and are meant to be modified, maybe a dynamic cronPattern config could be useful to set scheduled jobs on a go but this isn't available yet in this PR.

# Type of change


# Related services

- N/A

# How to test

:warning: **Important:  Make sure to make a backup of your data to test this feature**

1. Link PR volume or use image in loket
2. Add cleanup migrations through Loket https://github.com/lblod/app-digitaal-loket/pull/583
3. Let the service run it should delete resources at configured cronPattern
 
# What to check

- N/A

# Links to other PR's

- https://github.com/lblod/app-digitaal-loket/pull/583

# Notes

- N/A
